ui/src/main/webapp/lib/react-swatch/admin-palette.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react'
+
+import { Card, CardHeader } from 'material-ui/Card'
+import SelectField from 'material-ui/SelectField'
+import MenuItem from 'material-ui/MenuItem'
+
+import Swatch from './'
+import { getReadableTextColor } from './color-utils'
+
+export default class extends Component {
+  constructor (props) {
+    super(props)
+    this.state = ({ presetSelection: null })
+  }
+
+  changePreset (value) {
+    this.setState({ presetSelection: value })
+    this.props.setThemePreset(value)
+  }
+
+  render () {
+    const {
+      getTheme,
+      updateColor
+    } = this.props
+    const {
+      presetSelection
+    } = this.state
+
+    const cardStyle = ({
+      backgroundColor: getTheme().palette.backdropColor,
+      padding: '5px',
+      marginBottom: '10px'
+    })
+
+    const cardHeaderStyle = ({
+      margin: '0px',
+      fontSize: '18px',
+      whiteSpace: 'nowrap',
+      color: getReadableTextColor(getTheme().palette.backdropColor)
+    })
+
+    return (
+      <div style={{padding: '0px 5px'}}>
+        <Card style={cardStyle}>
```

Comment:
Might consider it in the future if there's ever a situation where one themeable component overlaps another in some meaningful way, but for now opacity just seems like a way to mix background colors in a gross way.

It's also hex-color unfriendly. (╯°□°)╯︵ ┻━┻